homework_2/task2.js

```diff
@@ -0,0 +1,24 @@
+const checkIfStart = str => {
+
+    if (str.length < 2) {
+        return false;
+    }
+    
+    let newString = str.substring(0, 2);
+    
+    if (
+        typeof newString === "string" &&
+        newString === "if"
+        ) {
+        return true;
+    }
+
+    else return false;
+}
+console.log(checkIfStart("if I could fly"))
+console.log(checkIfStart("iffy stuff"))
+console.log(checkIfStart("hello world"));
+
```

Comment:
Please think about how to show true if the string will be written in the upper case e.g. "If I could fly"